File: UserProfileService/UserProfileService.Application/DTOs/DishDTOs/UpdateDishDTO.cs
namespace UserProfileService.Application.DTOs
{
    public class UpdateDishDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string? Description { get; set; }
        public int AmountOfPortions { get; set; }

        public List<IngredientOfDishDTO> Ingredients { get; set; } = new List<IngredientOfDishDTO>();
    }
}

File: NutritionBalance.ClientFront/src/app/models/profile/DTOs/dish/update-dish.dto.ts
import { IngredientOfDishDTO } from './ingredient-of-dish.dto';

export class UpdateDishDTO {
  id!: number;
  name!: string;
  description?: string;
  amountOfPortions: number = 0;
  ingredients: IngredientOfDishDTO[] = [];
}
